.github/workflows/build-typescript-package.yml

```diff
@@ -1,7 +1,7 @@
 name: Build TypeScript Package
 
 on:
-  push:
+  pull_request:
```

Comment:
Changing this from `push` to `pull_request` since I want all PRs that change the paths listed below to run the TS build. With this being `push`, we were only getting the TS build running on commits that changed a file in one of the paths. With this update, it should continue to run this build on any PR that changes the TS package in any of its commits.